packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -750,7 +751,7 @@ oie.authenticator.oktaverify.method.totp.verify.enable.biometrics.point1 = 》Ý
 oie.authenticator.oktaverify.method.totp.verify.enable.biometrics.point2 = 》Öķţå Ṽéŕîƒý îš ûþ‐ţö‐ðåţé 홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.authenticator.oktaverify.method.totp.verify.enable.biometrics.point3 = 》Îñ Öķţå Ṽéŕîƒý، ƀîöɱéţŕîçš åŕé éñåƀļéð ƒöŕ ýöûŕ åççöûñţ ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.mobile = 》Ýöûŕ ŕéšþöñšé ŵåš ŕéçéîṽéð، ƀûţ ýöûŕ öŕĝåñîžåţîöñ ŕéǫûîŕéš ƀîöɱéţŕîçš· Ṁåķé šûŕé ýöûŕ ðéṽîçé šûþþöŕţš ƀîöɱéţŕîçš، Öķţå Ṽéŕîƒý îš ûþ‐ţö‐ðåţé åñð ƀîöɱéţŕîçš åŕé éñåƀļéð ƒöŕ ýöûŕ åççöûñţ îñ Öķţå Ṽéŕîƒý، ţĥéñ ţŕý åĝåîñ· Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
-oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.desktop = 》Ýöûŕ ŕéšþöñšé ŵåš ŕéçéîṽéð، ƀûţ ýöûŕ öŕĝåñîžåţîöñ ŕéǫûîŕéš ƀîöɱéţŕîçš· Ṁåķé šûŕé ýöûŕ ðéṽîçé šûþþöŕţš ƀîöɱéţŕîçš، Öķţå Ṽéŕîƒý îš ûþ‐ţö‐ðåţé، ƀîöɱéţŕîçš åŕé éñåƀļéð ƒöŕ ýöûŕ åççöûñţ îñ Öķţå Ṽéŕîƒý åñð ýöûŕ ðéṽîçé´š ƀîöɱéţŕîç šéñšöŕš åŕé åççéšîƀļé، ţĥéñ ţŕý åĝåîñ· ฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.desktop = 》Ýöûŕ ŕéšþöñšé ŵåš ŕéçéîṽéð، ƀûţ ýöûŕ öŕĝåñîžåţîöñ ŕéǫûîŕéš ƀîöɱéţŕîçš· Ṁåķé šûŕé ýöûŕ ðéṽîçé šûþþöŕţš ƀîöɱéţŕîçš، Öķţå Ṽéŕîƒý îš ûþ‐ţö‐ðåţé، ƀîöɱéţŕîçš åŕé éñåƀļéð ƒöŕ ýöûŕ åççöûñţ îñ Öķţå Ṽéŕîƒý åñð ýöûŕ ðéṽîçé´š ƀîöɱéţŕîç šéñšöŕš åŕé åççéššîƀļé، ţĥéñ ţŕý åĝåîñ· Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```

Comment:
add this key in `login.properties` as well..

Reply:
updated

Reply:
Looks like it was already [there](https://github.com/okta/okta-signin-widget/blob/c26adba7c4fd68233c50edc1a38a651b886fefb4/packages/%40okta/i18n/src/properties/login.properties#L1025), as result it was added when I generated the PL translations